conf/app.routes

```diff
@@ -5,6 +5,9 @@ GET         /assets/*file                 controllers.Assets.at(path="/public",
 
 # Pages
 
+GET        /:periodKey/honesty-declaration      controllers.HonestyDeclarationController.show(periodKey: String)
```

Comment:
Tiny thing but could you just indent the other controller paths here to be vertically in line with these two new ones